frontend/src/utils/TokenService.js

```
@@ -131,150 +134,176 @@ const TokenService = {
-      return Date.now() > parseInt(expiryTime);
+      if (!token) {
+          return true;
+      }
+
+      const expiryTimeStr = localStorage.getItem(this.tokenExpiryKey);
+      if (!expiryTimeStr) {
+           console.warn("Token expiry check: Expiry time missing.");
+           return true;
+      }
+
+      const expiryTime = parseInt(expiryTimeStr, 10);
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)